src/navigation/navigation.hpp

```diff
@@ -153,6 +154,10 @@ namespace navigation {
        * @brief Set the keyence used to fake, so the system knows to use central timestamps.
        */
       void setKeyenceFake();
+      /**
+       * @brief Write to file nav_data.csv
+       */
+      void logWrite();
```

Comment:
```suggestion
       * @brief Enable logging to file nav_data.csv
       */
      void logWrite();
```
The method doesn't actually write anything, right?